ignite/cmd/plugin.go

```diff
@@ -27,40 +27,43 @@ const (
 // A global variable is used so the list is accessible to the plugin commands.
 var plugins []*plugin.Plugin
 
-// LoadPlugins tries to load all the plugins found in configuration.
-// If no configuration found, it returns w/o error.
+// LoadPlugins tries to load all the plugins found in configurations.
+// If no configurations found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
-	localCfg, _, err := parseLocalConfig(rootCmd)
-	// if binary is run where there is no cfg, don't load
-	if err == nil {
-		plugins, err = plugin.Load(ctx, localCfg)
-		if err != nil {
-			return err
-		}
+	localCfg, err := parseLocalPlugins(rootCmd)
+	if err != pluginsconfig.ErrNotFound {
+		return nil
 	}
 
-	globalCfg, _, err := parseGlobalConfig()
-	// if binary is run where there is no cfg, don't load
-	if err == nil {
-		globalPlugins, err := plugin.Load(ctx, globalCfg)
-		if err != nil {
-			return err
-		}
-		plugins = append(plugins, globalPlugins...)
+	globalCfg, err := parseGlobalPlugins()
+	if err != pluginsconfig.ErrNotFound {
+		return nil
 	}
 
-	if len(plugins) == 0 {
+	// build list of all plugins
+	pluginsConfigs := globalCfg.Plugins
+	pluginsConfigs = append(pluginsConfigs, localCfg.Plugins...)
+	if len(pluginsConfigs) == 0 {
 		return nil
 	}
 
-	// TODO: sort duplicates
+	uniquePlugins := plugin.RemoveDuplicates(pluginsConfigs)
+	plugins, err = plugin.Load(ctx, uniquePlugins)
+	if err != nil || len(plugins) == 0 {
+		return nil
+	}
```

Comment:
Shouldn't the error be returned?

Reply:
https://github.com/ignite/cli/pull/3214/commits/f8500bf181e3f19e52be4c3403bd2993b576269c